examples/lammps/h2o-constraints/input.xml

```diff
@@ -1,56 +1,425 @@
-            </constraint>
-            <constraint mode="distance">
-                <atoms> [0, 1,
-                         0, 2]
-                </atoms>
-                <distances> [ 1.8897261, 1.8897261 ]  </distances>
-            </constraint>
-        </constraint>
-      </constrained_dynamics>
-    </motion>
-    <ensemble>
-      <temperature units='kelvin'> 300 </temperature>
-    </ensemble>
-  </system>
+<simulation mode='md' verbosity='high'>
+    <output prefix='simulation'>
+        <properties stride='1' filename='pot'>  [ pot_component_raw(0), pot_component_raw(1) ] </properties>
+        <properties stride='1' filename='out'>  [ step, time{picosecond}, conserved, temperature{kelvin}, kinetic_cv, potential, pressure_cv{megapascal} ] </properties>
+        <trajectory stride='1' filename='pos'>  positions{angstrom} </trajectory>
+        <checkpoint stride='10'/>
+    </output>
+    <total_steps> 100 </total_steps>
+    <prng>
+        <seed>31415</seed>
+    </prng>
+    <ffsocket name='lmp_short' mode='unix'>
+        <address>h2o-pimd+mts.1_short</address>
+    </ffsocket>
+    <ffsocket name='lmp_long' mode='unix'>
+        <address>h2o-pimd+mts.1</address>
+    </ffsocket>
+    <system>
+        <initialize nbeads='1'>
+            <file mode='pdb'> init.pdb </file>
+            <velocities mode='thermal' units='kelvin'> 300 </velocities>
+        </initialize>
+        <forces>
+            <!--
+             MTS setup - apply the fast (short-range) force in the inner loop and the slow (long-range) force in the outer loop.
+             Note that if the outer loop contains a *correction* to the inner loop the weights should be
+             [-1,1] (fast force) and [1,0] (slow force)
+             -->
+            <force forcefield='lmp_short'>
+                <mts_weights>[-1,1]</mts_weights>
+            </force>
+            <force forcefield='lmp_long'>
+                <mts_weights>[ 1,0]</mts_weights>
+            </force>
+        </forces>
+        <motion mode='constrained_dynamics'>
+            <constrained_dynamics mode='nvt' splitting='baoab'>
+                <!-- This uses a mixture of constraints, some with default target value initialisation
+                     and some with explicit specification; some molecules are fully fixed, some only
+                     have internal constraints, and some only have Eckart constraints -->
+                <nsteps_geo> [5] </nsteps_geo>
+                <csolver mode='sparse'>
+                    <maxit> 10000 </maxit>
+                    <norm_order> 2 </norm_order>
+                    <tolerance> .0001 </tolerance>
+                </csolver>
+                <nmts> [1, 4] </nmts>
+                <timestep units='femtosecond'> 4.0 </timestep>
+                <thermostat mode='langevin'>
+                    <tau units='femtosecond'> 5 </tau>
+                </thermostat>
+                <constraint mode="multi">
```

Comment:
@ceriottm : Would it be possible to create some sort of a template based input -- as is done in REMD -- so that the redundancy can be removed ?